packages/amplify-provider-awscloudformation/src/transform-graphql-schema.ts

```diff
@@ -54,14 +54,16 @@ const ROOT_APPSYNC_S3_KEY = 'amplify-appsync-files';
 const s3ServiceName = 'S3';
 
 export function searchablePushChecks(context, map): void {
-  const searchableModelTypes = Object.keys(map).filter(type => !map[type].includes('searchable') && map[type].includes('model'));
+  const searchableModelTypes = Object.keys(map).filter(type => map[type].includes('searchable') && map[type].includes('model'));
   if (searchableModelTypes.length) {
     const currEnv = context.amplify.getEnvInfo().envName;
     const teamProviderInfo = stateManager.getTeamProviderInfo();
     const apiCategory = teamProviderInfo[currEnv]?.categories?.api;
     const instanceType = apiCategory ? apiCategory[ResourceConstants.PARAMETERS.ElasticsearchInstanceType] : null;
     if (!instanceType || instanceType === 't2.small.elasticsearch') {
```

Comment:
@yuth will this check work in all regions?

Reply:
No there 2 regions where this will not work. It should be `t3.small.elasticsearch` in `me-south-1` and `eu-north-1`